include/bluetooth/gatt.h

```diff
@@ -725,6 +725,29 @@ struct _bt_gatt_ccc {
 			  const struct bt_gatt_attr *attr);
 };
 
+#if defined(CONFIG_BT_SETTINGS)
```

Comment:
Please avoid #ifdefs in header files, since a) it hides the doxygen documentation unless you add exceptions to documentation generation and b) it prevents usage of `IS_ENABLED()` in calling code.